src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -41,7 +44,7 @@ public static bool IsKind(this SyntaxTrivia trivia, SyntaxKind kind)
         /// <param name="node">The source node.</param>
         /// <param name="kind">The syntax kind to test for.</param>
         /// <returns><see langword="true"/> if the node is of the specified kind; otherwise, <see langword="false"/>.</returns>
-        public static bool IsKind(this SyntaxNode node, SyntaxKind kind)
+        public static bool IsKind([NotNullWhen(true)] this SyntaxNode? node, SyntaxKind kind)
```

Comment:
```suggestion
        public static bool IsKind([NotNullWhen(returnValue: true)] this SyntaxNode? node, SyntaxKind kind)
```

It reads a bit better IMHO.

Reply:
CC @sharwell 

I have no real opinion on which way we should do this but I've seen both styles. My only preference is we pick one and stick with it. Happy to do either but largely I've been doing the current style.

Reply:
I used to put `returnValue:` but I eventually stopped.

Reply:
For now sticking with `NotNullWhen(true)]`. More than happy though to have a convo on what the preferred style is and help move the code base in that direction.

---
In reply to: [362651475](https://github.com/dotnet/roslyn/pull/40699#discussion_r362651475) [](ancestors = 362651475)